cell_classification/application_test.py

```diff
@@ -0,0 +1,35 @@
+from application import CellClassification
+from model_builder import ModelBuilder
+from segmentation_data_prep_test import prep_object_and_inputs
+import numpy as np
+import tempfile
+import toml
+import os
+
+
+def predict_test():
```

Comment:
We should have a separate test for the normalize function

Reply:
Agree!